controller/controller_server/csi_controller_server.py

```diff
@@ -477,14 +471,14 @@ def ControllerGetCapabilities(self, request, context):
         logger.info("finished ControllerGetCapabilities")
         return res
 
-    def __get_identity_config(self, attribute_name):
+    def get_identity_config(self, attribute_name):
         return self.cfg['identity'][attribute_name]
 
     @handle_common_exceptions(csi_pb2.GetPluginInfoResponse)
     def GetPluginInfo(self, _, context):
         logger.info("GetPluginInfo")
-        name = self.__get_identity_config("name")
-        version = self.__get_identity_config("version")
+        name = self.get_identity_config("name")
```

Comment:
why we are not getting the `self.cfg['identity']` once (maybe in init) and then "extracting" the info from there?

Reply:
sounds like a good idea for a different ticket

Reply:
as I recall our way of work is to fix and approve code where ever the PR is reaching but if we are in a problem with time here please open a ticket for it and I will resolve

Reply:
We're not in a problem with time here since we waste a lot of time by commenting, opening tickets, etc.
Changing the code is faster in simple cases like this one IMO.

Reply:
CSI-3238

Reply:
we're in a problem with time here because we are close to code freeze.
and commenting, opening tickets etc... is faster than increasing the diff, which potentially increases the amount of comments, tickets to open, etc...
in addition, the context of this whole PR is already a "collateral" change for the main ticket, so we shouldn't have "collateral" changes to the "collateral" changes in the same ticket, it has got to end somewhere